MarkBind/markbind
# Remove override of `title` in the init site

**What is the purpose of this pull request?**




**Overview of changes:**
This PR resolves #242.

While the original issue was opened before PR #935 which overhauled the site created on `markbind init` to the template currently being used, the issue was not closed, even though the main issue of duplicate declaration of `title` is no longer a problem, as the `title` in `index.md` was removed in PR #935.

Modifications were made as per the suggestion made in the original issue to have the starter site use the typical technique (i.e. `title` is declared within the `frontmatter` tag in the `.md` file) instead of declaring it in `site.json`, as declaring the `title` in `site.json` will override the one declared within the `frontmatter` tag. 

Once it's been checked that no documentation needs to be updated, the [WIP] will be removed from the title of the PR.

**Anything you'd like to highlight/discuss:**

**Testing instructions:**

Running `markbind init` to create a new site (after binding the MarkBind version to the one containing changes made in this PR) will create an `index.md` now containing `title` in its `frontmatter` tag, and a `site.json` that no longer declares the `title`.



**Proposed commit message: (wrap lines at 72 characters)**
Remove override of title property in the init site


---

**Checklist:** :ballot_box_with_check:
